hw/bsp/nucleo-l476rg/src/hal_bsp.c

```diff
@@ -79,7 +93,39 @@ static struct stm32_hal_i2c_cfg i2c_cfg0 = {
     .hic_pin_scl = MCU_GPIO_PORTB(8),     /* PB8 - D15 on CN5 */
     .hic_pin_af = GPIO_AF4_I2C1,
     .hic_10bit = 0,
-    .hic_timingr = 0x10420F13,            /* 100KHz at 8MHz of SysCoreClock */
+    .hic_timingr = 0x00000E14,            /* 100KHz at 4MHz of SysCoreClock */
+};
+#endif
+
+#if MYNEWT_VAL(I2C_1)
+static struct stm32_hal_i2c_cfg i2c_cfg1 = {
+    .hic_i2c = I2C2,
+    .hic_rcc_reg = &RCC->APB1ENR1,
+    .hic_rcc_dev = RCC_APB1ENR1_I2C2EN,
+    .hic_pin_sda = MYNEWT_VAL(I2C_1_PIN_SDA),
+    .hic_pin_scl = MYNEWT_VAL(I2C_1_PIN_SCL),
```

Comment:
Since you added it to syscfg, any reason for not having updated the other 2 with similar `I2C_X_PIN_` naming?

Reply:
that is right, force pushed changes that cover this